SPCTR-3444: Fix - Buttons margin getting affected by child margin issue (#3550)
brainstormforce/ultimate-addons-for-gutenberg: README.md

### x.x.x ###
* Improvement: Buttons - Added a new vertical alignment option.

> * Fix: Buttons - Resolved an issue in which the margins added to buttons is getting overridden by child-button margins. 

Reviewer: ```suggestion
* Fix: Buttons - Resolved an issue in which the margins added to buttons group would get overridden by child button margins. 
```

Author: Done.
